snuba/subscriptions/combined_scheduler_executor.py

```diff
@@ -180,6 +180,8 @@ def __init__(
         self.__executor_factory = SubscriptionExecutorProcessingFactory(
             max_concurrent_queries,
             total_concurrent_queries,
+            # total_partition_count should always be 1 when using the combined executor/scheduler
+            1,
```

Comment:
I don't think this is a safe assumption. See https://github.com/getsentry/snuba/issues/2884